bHY6yh-d1e792/cod-klc89d_bHY6yh-d1e792.xml

```diff
@@ -106,7 +106,7 @@
               <lb ed="#Ka" break="no"/>telligere potest viam suam</quote>
             </p>
             <p xml:id="bHY6yh-d1e792-d1e200"> 
-              <quote xml:id="bHY6yh-d1e792-Qd1e202">omitted</quote> <!--"A domino diriguntur gressus viri: quis autem hominum intellige re potest viam suam" in quote above-->
+              omitted<quote xml:id="bHY6yh-d1e792-Qd1e202"></quote> <!--"A domino diriguntur gressus viri: quis autem hominum intellige re potest viam suam" in quote above-->
```

Comment:
@graydonlee if the entire entry is omitted, you can comment out that quote element as well.

So here you could just say:

```xml
omittted <!-- <quote>....</quote> -->
```